backEnd/src/app.controller.ts

```diff
@@ -1,9 +1,25 @@
-import { Controller, Get } from '@nestjs/common';
+import { Controller, Get, Request, Post, UseGuards  } from '@nestjs/common';
 import { AppService } from './app.service';
+import { LocalAuthGuard } from 'authentication/local-auth.guard';
+import { AuthenticationService} from 'authentication/authentication.service';
+import { JwtAuthGuard } from 'authentication/jwt-auth.guard';
+import { Public } from 'decorators/public.decorator';
 
 @Controller()
 export class AppController {
-  constructor(private readonly appService: AppService) {}
+  constructor(
+              private readonly appService: AppService,
+              private authService: AuthenticationService
+              ){}
+
+
+//I dont know where I Can put the login method.              
+  @UseGuards(LocalAuthGuard)
+  @Public()
```

Comment:
We don't need the Public decorator here because the local auth guard needs to authenticate